model/Software/Vocabularies/FileKindType.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# FileKindType
+
+## Summary
+
+Enumeration of the different kinds of SPDX file.
+
+## Description
+
+An SPDX file may represent a file on disk or a directory of files.  In the future, this may be extended to other kinds (e.g. network based files).
+
+## Metadata
+
+- name: FileKindType
+
+## Entries
+
+- file: The file represents a single file.
```

Comment:
Can we make it explicit, that if the field is not specified, the assumption is that fileKind is "file".